synapse/rest/client/v2_alpha/keys.py

```diff
@@ -227,8 +229,46 @@ def on_POST(self, request):
         defer.returnValue((200, result))
 
 
+class SigningKeyUploadServlet(RestServlet):
+    """
+    POST /keys/device_signing/upload HTTP/1.1
+    Content-Type: application/json
+
+    {
+    }
+    """
+
+    PATTERNS = client_patterns("/keys/device_signing/upload$")
```

Comment:
is this meant to be on `r0` or `unstable`? please can it only be on one of them, anyway?

Reply:
I guess it goes on `unstable` until the MSC goes through?

Reply:
yeah, I think so.